[2024-08-05T09:09:49.481+0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-05T09:09:49.515+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Nairobi scheduled__2024-08-04T08:00:00+00:00 [queued]>
[2024-08-05T09:09:49.528+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Nairobi scheduled__2024-08-04T08:00:00+00:00 [queued]>
[2024-08-05T09:09:49.529+0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-05T09:09:49.559+0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_air_pollution_data_Nairobi> on 2024-08-04 08:00:00+00:00
[2024-08-05T09:09:49.566+0300] {standard_task_runner.py:64} INFO - Started process 16276 to run task
[2024-08-05T09:09:49.570+0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'air_pollution_data_dag', 'fetch_air_pollution_data_Nairobi', 'scheduled__2024-08-04T08:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/air_pollution_data_dag.py', '--cfg-path', '/tmp/tmpa2cqm9w8']
[2024-08-05T09:09:49.571+0300] {standard_task_runner.py:91} INFO - Job 75: Subtask fetch_air_pollution_data_Nairobi
[2024-08-05T09:09:49.616+0300] {task_command.py:426} INFO - Running <TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Nairobi scheduled__2024-08-04T08:00:00+00:00 [running]> on host std21092
[2024-08-05T09:09:49.722+0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='air_pollution_data_dag' AIRFLOW_CTX_TASK_ID='fetch_air_pollution_data_Nairobi' AIRFLOW_CTX_EXECUTION_DATE='2024-08-04T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-04T08:00:00+00:00'
[2024-08-05T09:09:49.724+0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-05T09:09:51.692+0300] {logging_mixin.py:188} INFO - Erreur lors de la récupération des données pour Nairobi
[2024-08-05T09:09:51.693+0300] {python.py:237} INFO - Done. Returned value was: None
[2024-08-05T09:09:51.693+0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-05T09:09:51.701+0300] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=air_pollution_data_dag, task_id=fetch_air_pollution_data_Nairobi, run_id=scheduled__2024-08-04T08:00:00+00:00, execution_date=20240804T080000, start_date=20240805T060949, end_date=20240805T060951
[2024-08-05T09:09:51.748+0300] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-05T09:09:51.765+0300] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-05T09:09:51.769+0300] {local_task_job_runner.py:222} INFO - ::endgroup::
